src/dashboard/src/templates/archival_storage/list.html

```diff
@@ -96,7 +96,11 @@
       <tbody>
         {% for item in aips %}
           <tr>
-            <td><a href="{% url 'view_aip' item.uuid %}">{{ item.name }}</a></td>
+              {% if item.encrypted %}
+              <td><a href="{% url 'view_aip' item.uuid %}">{{ item.name }} ({% trans "encrypted" %})</a></td>
```

Comment:
I think it's better to do as follows:

```django
{% blocktrans with aip_name=item.name %}{{ aip_name }} encrypted{% endblocktrans %}
```

With that the translator sees `{{ aip_name }} encrypted` and they can freely move the placeholder and fully control the structure of the sentence.